adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala

```diff
@@ -392,6 +386,46 @@ sealed abstract class NucleotideContigFragmentRDD extends AvroGenomicRDD[Nucleot
   }
 
   /**
+   * From a set of contigs, returns list of sequences based on reference regions provided
+   * @param regionsList List of Reference regions over which to get sequences
+   * @return RDD[(ReferenceRegion, String)] of region -> sequence pairs
+   */
+  def extractRegions(regionsList: List[ReferenceRegion]): RDD[(ReferenceRegion, String)] = {
```

Comment:
We generally prefer to expose `Iterable` as the collection type in public method signatures.